CHANGELOG.md

```
@@ -1,5 +1,22 @@
 # Changelog
 
+## [v1.0.0]()
```

markdownlint-cli2 (0.22.0)

[warning] No empty links

(MD042, no-empty-links)